index.html

```diff
@@ -55,5 +55,146 @@ <h1> Hey there. I’m Alvin <br>
   <button class="btn arrow"> <i class="fa fa-angle-down"> </i></button>
   </section>
   </section>
+
+  <section class="works_section">
+  <div class="recent_works_title">
+  <h1>My Recent Works</h1>
+  </div>
+  <div class="cards">
+  <article class="card">
+  <div class="sane">
+  </div>
+  <div class="content">
+  <h1>Multi-post Stories <br>Gain + Glory</h1>
+  <ul class="category">
+  <li><a href="#">Ruby on Rails</a></li>
+  <li><a href="#">Css</a></li>
+  <li><a href="#">Javascript</a></li>
+  <li><a href="#">Html</a></li>
+  </ul>
+   <div class="open"><a href="#">See Project</a></div>
+  </div>
+  </article>
+  <article class="card">
+  <div class="sane"></div>
+  <div class="content">
+  <h1>Multi-post Stories <br>Gain + Glory</h1>
+  <ul class="category">
+  <li><a href="#">Ruby on Rails</a></li>
+  <li><a href="#">Css</a></li>
+  <li><a href="#">Javascript</a></li>
+  <li><a href="#">Html</a></li>
+  </ul>
+  <div class="open"><a href="#">See Project</a></div>
+  </div>
+  </article>
+  <article class="card">
+  <div class="sane">
+  </div>
+  <div class="content">
+  <h1>Multi-post Stories <br>Gain + Glory</h1>
+  <ul class="category">
+  <li><a href="#">Ruby on Rails</a></li>
+  <li><a href="#">Css</a></li>
+  <li><a href="#">Javascript</a></li>
+  <li><a href="#">Html</a></li>
+  </ul>
+  <div class="open"><a href="#">See Project</a></div>
+  </div>
+  </article>
+  <article class="card">
+  <div class="sane">
+  </div>
+  <div class="content">
+  <h1>Multi-post Stories <br>Gain + Glory</h1>
+  <ul class="category">
+  <li><a href="#">Ruby on Rails</a></li>
+  <li><a href="#">Css</a></li>
+  <li><a href="#">Javascript</a></li>
+  <li><a href="#">Html</a></li>
+  </ul>
+  <div class="open"><a href="#">See Project</a></div>
+  </div>
+  </article>
+  <article class="card">
+  <div class="sane">
+  </div>
+  <div class="content">
+  <h1>Multi-post Stories <br>Gain + Glory</h1>
+  <ul class="category">
+  <li><a href="#">Ruby on Rails</a></li>
+  <li><a href="#">Css</a></li>
+  <li><a href="#">Javascript</a></li>
+  <li><a href="#">Html</a></li>
+  </ul>
+  <div class="open"><a href="#">See Project</a></div>
+  </div>
+  </article>
+  <article class="card">
+  <div class="sane">
+  </div>
+  <div class="content">
+  <h1>Multi-post Stories <br>Gain + Glory</h1>
+  <ul class="category">
+  <li><a href="#">Ruby on Rails</a></li>
+  <li><a href="#">Css</a></li>
+  <li><a href="#">Javascript</a></li>
+  <li><a href="#">Html</a></li>
+  </ul>
+  <div class="open"><a href="#">See Project</a></div>
+  </div>
+  </article>
+  </div>
+</section>
+
+<section class="about">
+  <h1>About</h1>
+  <div class="about_info">
+  <p> I can help you build a product , 
+  feature or website Look through some 
+  of my work and experience! If you like 
+  what you see and have a project you need coded, 
+  don’t hestiate to contact me.</p>
+  </div>
+  <div class="resume"><a href="#">Get My Resume</a></div>
```

Comment:
- [ ] Please The button `Get my resume` follows the button styling: Active, Hover, and Pressed (see design)

![image](https://user-images.githubusercontent.com/63972670/159131134-f8c13d78-9330-408f-b0cf-cb62a9959614.png)